src/extension/module.js

```diff
@@ -1546,6 +1572,30 @@
             },
           },
         }));
+        buttonGroup.append(createTag({
+          tag: 'button',
+          text: i18n(sk, ok.length === 1 ? 'open_url' : 'open_urls'),
+          lstnrs: {
+            click: (evt) => {
+              evt.stopPropagation();
+              if (ok.length <= 20 || window.confirm(i18n(sk, 'open_urls_confirm').replace('$1', ok.length))) {
+                ok.forEach((item) => {
+                  const url = `https://${host}${item.path}`;
+                  const [{ viewer } = {}] = findViews(sk, VIEWS.CUSTOM, item.path);
+                  if (viewer) {
+                    const viewUrl = new URL(viewer, url);
+                    viewUrl.searchParams.set('url', url);
```

Comment:
This is still using the `url` query param, not sure why you ignored my comment here... ?

Reply:
Sorry I missed that